src/main/java/org/ibp/api/brapi/v2/inventory/TransactionDto.java

```diff
@@ -0,0 +1,70 @@
+package org.ibp.api.brapi.v2.inventory;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.Date;
+import java.util.Map;
+
+@AutoProperty
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TransactionDto {
+
+	private Map<String, Object> additionalInfo;
+	private Double amount;
+	private String transactionDescription;
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyyMMdd")
```

Comment:
Date format should be yyyy-MM-ddThh:mm:ssz. Based on this spec: https://github.com/plantbreeding/API/blob/master/Specification/GeneralInfo/Date_Time_Encoding.md